.gitignore

```diff
@@ -1,2 +1,3 @@
 _site
-.sass-cache
```

Comment:
Sugeruję zostawić .sass-cache w `.gitignore` i używać `@import`ów w `css/main.scss` tak jak to wcześniej było robione - to łączy wszystkie pliki w jeden, zmniejszając liczbę zapytań.